src/libsyntax/attr.rs

```diff
@@ -77,6 +81,93 @@ pub fn is_used(attr: &Attribute) -> bool {
     })
 }
 
+pub trait AttrMetaListItemMethods {
```

Comment:
My opinion is that since this is not the way `name()` is used in the code, it would be error prone to make this change.